addons/mail/static/tests/qunit_suite_tests/components/message_tests.js

```diff
@@ -835,6 +835,58 @@ QUnit.test('chat with author should be opened after clicking on their avatar', a
     );
 });
 
+QUnit.test('chat with author should be opened after clicking on their name', async function (assert) {
+    assert.expect(4);
+
+    const pyEnv = await startServer();
+    const resPartnerId = pyEnv['res.partner'].create({});
+    pyEnv['res.users'].create({ partner_id: resPartnerId });
+    pyEnv['mail.message'].create({
+        author_id: resPartnerId,
+        body: 'not empty',
+        model: 'res.partner',
+        res_id: resPartnerId,
+    });
+    const views = {
+        'res.partner,false,form':
+            `<form string="Partners">
+                <div class="oe_chatter">
+                    <field name="message_ids"/>
+                </div>
+            </form>`,
+    };
```

Comment:
There is now a default view provided, this should no longer be necessary.

Reply:
At the time I added this `openView` it wasn't working without adding views explicitly. But, somehow now it's working. So, I have removed this extra part.